examples/megacli.cpp

```diff
@@ -4234,7 +4234,7 @@ void exec_open(autocomplete::ACState& s)
             clientFolder->logout();
         }
 
-        return clientFolder->app->login_result(clientFolder->folderaccess(s.words[1].s.c_str()));
+        return clientFolder->app->login_result(clientFolder->folderaccess(s.words[1].s.c_str(), nullptr));
```

Comment:
I've added support for `login`. It would be great if you add support for `export`, and also check codepaths that look blocked in the SDK for writable public links, such as movements (renames, indeed).